beacon_node/eth2_libp2p/src/discovery/mod.rs

```diff
@@ -217,7 +219,50 @@ impl<TSpec: EthSpec> Discovery<TSpec> {
             EventStream::InActive
         };
 
-        // Obtain the event stream
+        // get futures for requesting the Enrs associated to these MultiAddrs and wait for their
+        // completion
+        let mut fut_coll = config
+            .boot_nodes_multiaddr
+            .iter()
+            .map(|addr| addr.to_string())
+            // request the ENR for this multiaddr and keept the original for logging
+            .map(|addr| {
+                futures::future::join(
+                    discv5.request_enr(addr.clone()),
+                    futures::future::ready(addr),
+                )
+            })
+            .collect::<FuturesUnordered<_>>();
+
+        while let Some((result, original_addr)) = fut_coll.next().await {
```

Comment:
This could potentially take some time, depending on the number of boot nodes. 
As we currently stop all of lighthouse from progressing, I think we should inform the user that we are attempting to connect to the bootnodes. 
Something like a:
```rust
info!(log, "Contacting boot-nodes for their ENR")
```
Only if `!fut_coll.is_empty()`